runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatOverlay.java

```diff
@@ -88,11 +93,78 @@ public Dimension render(Graphics2D graphics)
 			}
 
 			tooltipManager.add(new Tooltip(b.toString()));
+			return null;
+		}
+
+		final ItemComposition item = itemManager.getItemComposition(entry.getIdentifier());
+		if (item != null)
+		{
+			final ItemStatBonuses.ItemStats stats = statBonuses.getBonus(item);
+
+			if (stats != null)
+			{
+				tooltipManager.add(new Tooltip(buildStatBonusString(stats)));
+			}
 		}
 
 		return null;
 	}
 
+	private String getChangeString(final String label, final double value, final boolean inverse)
+	{
+		final Color plus = Positivity.getColor(config, Positivity.BETTER_UNCAPPED);
+		final Color minus = Positivity.getColor(config, Positivity.WORSE);
+
+		if (value == 0)
+		{
+			return "";
+		}
+
+		final Color color;
+
+		if (inverse)
+		{
+			color = value > 0 ? minus : plus;
+		}
+		else
+		{
+			color = value > 0 ? plus : minus;
+		}
+
+		final String valueString = (int)value == value ? String.valueOf((int)value) : String.valueOf(value);
+		return label + ": " + ColorUtil.wrapWithColorTag(valueString, color) + "</br>";
+	}
+
+	private String buildStatBonusString(ItemStatBonuses.ItemStats s)
+	{
+		StringBuilder b = new StringBuilder();
+		b.append(getChangeString("Weight", s.getWeight(), true));
+
+		final ItemStatBonuses.ItemEquipmentStats e = s.getEquipment();
+		if (s.isEquipable() && e != null)
+		{
+			b.append(getChangeString("A Stab", e.getAstab(), false));
+			b.append(getChangeString("A Slash", e.getAslash(), false));
+			b.append(getChangeString("A Crush", e.getAcrush(), false));
+			b.append(getChangeString("A Magic", e.getAmagic(), false));
+			b.append(getChangeString("A Range", e.getArange(), false));
+
+			b.append(getChangeString("D Stab", e.getDstab(), false));
+			b.append(getChangeString("D Slash", e.getDslash(), false));
+			b.append(getChangeString("D Crush", e.getDcrush(), false));
+			b.append(getChangeString("D Magic", e.getDmagic(), false));
+			b.append(getChangeString("D Range", e.getDrange(), false));
+
+			b.append(getChangeString("Melee Str", e.getStr(), false));
+			b.append(getChangeString("Range Str", e.getRstr(), false));
+			b.append(getChangeString("Magic Dmg", e.getMdmg(), false));
```

Comment:
This is a percentage so I feel like it should show a percentage sign in the tooltip.

Reply:
This is no longer an issue